utils/args_parser.go

```diff
@@ -183,6 +183,15 @@ func (p *ArgsParser) HasReceived(name string) bool {
 	return found && len(f.values) > 0
 }
 
+func (p *ArgsParser) HasReceivedOneOf(names []string) bool {
```

Comment:
Might be too much to open the ArgsParser API up like this. It felt more natural to put this logic here instead of the Issue. Let me know if you want me to move it.

Reply:
I think this is too specific for ArgsParser (which I intentionally kept small), so I'd rather that this is inlined in issue.go or encapsulated as a function that takes an ArgsParser instance and a list of flags.

Btw you can have a Go function accept a list of arguments instead of a slice:

```go
func HasReceivedOneOf(names ...string) bool {
}

// usage:
HasReceivedOneOf("foo", "bar")
```

Reply:
> I think this is too specific for ArgsParser (which I intentionally kept small), so I'd rather that this is inlined in issue.go or encapsulated as a function that takes an ArgsParser instance and a list of flags.

Sounds good, I inlined the check in issue.go.
 
> Btw you can have a Go function accept a list of arguments instead of a slice:
 
Cool, I didn't think of that. Thanks for the tip!